### ISSUE TITLE
Does not work with `<testsuite>` in `<testsuite>`

### ISSUE THREAD

**Issue by dishmaker** [borisfaure/junit-parser#9]
```rust
use std::io::Cursor;

fn main() {
    if let Err(err) = run() {
        println!("err: {:?}", err);
    }
}

fn run() -> eyre::Result<()> {
    let xml = r#"
<testrun name="mytestrun" tests="3" failures="1">
    <testsuite name="firstsuite">
        <testsuite name="mygroup">
            <testcase classname="foo1" name="a"/>
            <testcase classname="foo2" name="b"/>
        </testsuite>
        <testcase classname="foo3" name="c"/>
        <testcase classname="foo4" name="d"/>
        <testcase classname="foo5" name="e"/>

        <testsuite name="mygroup2">
            <testcase classname="foo6" name="f"/>
            <testcase classname="foo7" name="g"/>
        </testsuite>
    </testsuite>
</testrun>
"#;
    let cursor = Cursor::new(xml);
    let parsed = junit_parser::from_reader(cursor)?;
    
    for suite in parsed.suites {
        println!("suite: {:?}", suite.name);

        for test in suite.cases {
            println!("case: {:?}", test.name);
        }
    }
    
    Ok(())
}
```

prints:
```
suite: "firstsuite"
case: "foo1::a"
case: "foo2::b"
```

**Comment by dishmaker** [comment]
Older version: JUnit 4.13.2 generates such reports.

**Comment by borisfaure** [comment]
I've fixed this in [junit-parser v1.1.0](https://github.com/borisfaure/junit-parser/releases/tag/v1.1.0)

**Comment by dishmaker** [comment]
Thanks!
